pkg/netutil/cidr.go

```diff
@@ -3,9 +3,37 @@ package netutil
 import (
 	"net"
 
+	"github.com/stackrox/rox/pkg/logging"
 	"github.com/stackrox/rox/pkg/utils"
 )
 
+var (
+	privateSubnets = []*net.IPNet{
```

Comment:
Imho it would be good to consolidate this list somewhere in the codebase, as it occurs multiple times (with some differences). The values in `pkg/net/addr.go` are probably a good starting point.

Reply:
Took a shoot at refactoring this. To avoid keeping too many different subsets the insecure_registries checks (`central/compliance/checks/common/insecure_registries.go` and `pkg/compliance/checks/common/insecure_registries.go`) are now also checking private IPv4 addresses mapped to IPv6 values. Let me know what do you think.